### DISCUSSION TITLE
[QUESTION] Self Hosting ?

### DISCUSSION THREAD

**Discussion by monish-1234** [AndresMorelos/InvoncifyReleases#24] [Q&A] [answered]
Hey !

Since this basically runs on Javascript, Is it possible to use this as a webapp ? 
Since I change between devices frequently, I want it to run on my server (aws, hostiger) and then be accessible via port 80 ? 
We can just access control via vpn or something else from the server end for security or have a login setup while installing maybe ?

Please let me know your thoughts on this. 

And having a Discord channel might help for questions like this 😬 

Thanks ! 
Sorry if this was a weird question

**Comment by AndresMorelos** [comment, marked as answer]
Hello, @monish-1234.

Well, I never thought to move this app to a webapp, but, currently, I'm developing an API in order to offer sync between devices, It can take some weeks since I need to make tests for security and stability.

If you wanna make this code a webapp feel free to take the code and deploy it as a react website.
